frontend/src/components/Search/SimpleSearch/SimpleSearch.logic.ts

```diff
@@ -0,0 +1,49 @@
+import type { QueryParams } from "~/stores/searchParams";
+import { DocumentKind } from "~/types";
+
+export function getUrl(category?: string) {
```

Comment:
It might be a good idea to always add explicitly the return type to the function